tools/marvin/marvin/config/test_data.py

```diff
@@ -121,7 +121,7 @@
             "name": "LargeInstance",
             "displaytext": "LargeInstance",
             "cpunumber": 1,
-            "cpuspeed": 500,
+            "cpuspeed": 1024,
```

Comment:
while appending to this list could be fairly non fragile to tests, editing existing entries can be extremely fragile to other tests that are using it. We need to be sure this does not brake any of the smoketests, since they are our main source of verifying PRs.